Cargo.toml

```
@@ -20,8 +20,8 @@ default = ["macro-check"]
 macro-check = ["actix-grants-proc-macro"]
 
 [dependencies]
-actix-web = { version = "3.0.0", default-features = false}
-actix-grants-proc-macro = { version = "1", optional = true }
+actix-web = { version = "4.0.0-beta.3", default-features = false }
+actix-grants-proc-macro = { path = "./proc-macro", optional = true }
```

The version is required for cases of crate publication.
The Cargo book suggests specifying both parameters in such cases.
Before release, we will just bump the version.

```suggestion
actix-grants-proc-macro = { path = "./proc-macro", version = "1", optional = true }
```